src/auth/oidc-broker.ts

```diff
@@ -53,11 +53,15 @@ export const initializeClientId = _.memoize(async (): Promise<void> => {
   oidcStore.update((state) => ({ ...state, config: oidcConfig }));
 });
 
+export const initializeOidcUserManager = () => {
+  const userManager: UserManager = new UserManager(getOidcConfig());
+  oidcStore.update((state) => ({ ...state, userManager }));
+};
+
 // Instantiate a UserManager directly to populate the logged-in user at app initialization time.
 // All other auth usage should use the AuthContext from oidcStore.
 export const getCurrentOidcUser = async (): Promise<OidcUser | null> => {
-  const userManager: UserManager = new UserManager(getOidcConfig());
-  return userManager.getUser();
+  return oidcStore.get().userManager!.getUser();
```

Comment:
what does the ! do here I forget

Reply:
It ignores that this value could be null. In this case it should not be since it is populated on App creation, so I ignore it to satisfy the types